### DISCUSSION TITLE
Update pip dependencies using VCS as version

### DISCUSSION THREAD

**Discussion by simao-silva** [renovatebot/renovate#15703] [Request Help]
Hi,
I am using Renovate 32.64.4 self-hosted and I am trying to set up Renovate to update my requirements.txt using pip in the following format 
```
-e git+https://git.example.com/MyProject.git@da39a3ee5e6b4b0d3255bfef95601890afd80709egg=pkg
```

I've search the discussions and the documentation bit did not find a solution. Does Renovate have the ability to parse this format type?

**Comment by rarkins** [comment]
You can see at the bottom of https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/pip_requirements/extract.spec.ts there's some git package tests but I don't see one matching closely to what you have

**Comment by simao-silva** [comment]
So, this format is not supported?
